src/initializers/kafka/kafka.ts

```diff
@@ -35,6 +34,7 @@ export default class OrkaKafka {
       authenticationTimeout
     });
 
+    if (options && !options.createPartitioner) options.createPartitioner = Partitioners.DefaultPartitioner;
```

Comment:
Can we add the ability to define the partitioner in docs too? 
https://github.com/Workable/orka/blob/master/docs/integrations/kafka.md